docs/en/setup/service-agent/java-agent/TLS.md

```diff
@@ -19,6 +19,10 @@ Only support **no mutual auth**.
 ### Agent config
 - Place `ca.crt` into `/ca` folder in agent package. Notice, `/ca` is not created in distribution, please create it by yourself.
 
-Agent open TLS automatically after the `/ca/ca.crt` file detected.
+Agent will open TLS upon the following config value. If it's true, TLS will be forced to use, and TLS works by default.
+At the same time, a trust manager will be constructed for TLS authentication once the `/ca/ca.crt` file detected.
+```
+agent.is_grpc_channel_tls_forced=${SW_AGENT_IS_GRPC_CHANNEL_TLS_FORCED:true}
+```
```

Comment:
Also, there is a config table in the agent setup doc, please update it too.

Reply:
What's the exact file name ?

Reply:
This one, https://github.com/apache/skywalking/blob/master/docs/en/setup/service-agent/java-agent/README.md#table-of-agent-configuration-properties